modules/tinymce/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Dialogs will not exceed the window height on smaller screens #TINY-8146
 - Text alignment could not be applied to `pre` elements #TINY-7715
+- Dialogs would in some cases not prevent key presses from escalating to listeners #TINY-7005
```

Comment:
Do we need to be more specific here that it's the keyup events that would be triggered for Esc and Enter?

Reply:
Hmm... technically this applies to many things. I don't think we should be saying `Dialogs` specifically here, as this could also apply to other UI components. I do agree we should say this applies to `Esc` only though. Maybe something like this?
```suggestion
- UI components, such as dialogs, would in some cases cause the `esc` keyup event to incorrectly trigger inside the editor #TINY-7005
```